### ISSUE TITLE
Remove unused location from nginx

### ISSUE THREAD

**Issue by pedrobmarin** [bigbluebutton/bigbluebutton#11434]
The HTML5 client is being packed with an nginx location that seems to be unused

```
location /_timesync {
  proxy_pass http://127.0.0.1:3000;
}
```

After some further investigation I found out that this location is normally used along side with this Meteor plugin https://github.com/Meteor-Community-Packages/meteor-timesync that the HTML5 client used to install but removed in https://github.com/bigbluebutton/bigbluebutton/commit/a8c1df7f22badfbeb415058b7bb1a3390e31ba02#diff-d9ed3391e0315fb6999cac11cb62796d0e9f0e53ef51e9af50aa4e00547a09e0L11

**Comment by antobinary** [comment]
I have removed this nginx block from bbb-html5.nginx and the change will be included in 2.3-alpha7

**Comment by antobinary** [comment]
This change was released with https://github.com/bigbluebutton/bigbluebutton/releases/tag/v2.3-alpha-7
